src/apps/engine/src/core_impl.cpp

```diff
@@ -238,7 +238,7 @@ bool CoreImpl::Initialize()
 
 void CoreImpl::ProcessEngineIniFile()
 {
-    char String[_MAX_PATH];
+    char String[MAX_PATH];
```

Comment:
I'm sure, that `MAX_PATH` == `_MAX_PATH` on Windows. So it will be easier to fix it for Linux in future.
In fact on Linux there is only `#define PATH_MAX 4096` in `limits.h`.